src/HDInsight/HDInsight/Models/Management/AzureHDInsightRegionsCapability.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.Azure.Management.HDInsight.Models;
```

Comment:
@aim-for-better nit: please move these `using` statements down below with the other `using` statements

Reply:
> @aim-for-better nit: please move these `using` statements down below with the other `using` statements

fixed